settings.yml

```diff
@@ -0,0 +1,13 @@
+---
+project: xscope_fileio
+title: xscope_fileio
+version: 0.2.1
+
+documentation:
+  exclude_patterns_path: doc/exclude_patterns.inc
+  doxygen_projects:
+    xscope_fileio:
+      doxyfile_path: doc/Doxyfile.inc
+      doxy_overrides: |
+        GENERATE_HTML = yes
+        GENERATE_LATEX = yes
```

Comment:
is there a reason we keep generating doxygen output directly? As far as I know, the output of this is not accessible unless xmosdoc is run in dev mode to keep intermediate files around.